helm/provider.go

```diff
@@ -381,13 +384,33 @@ func (m *Meta) GetHelmConfiguration(namespace string) (*action.Configuration, er
 	defer m.Unlock()
 
 	actionConfig := new(action.Configuration)
-	if err := actionConfig.Init(m.ConfigFlags, namespace, m.HelmDriver, debug); err != nil {
+	cf := getKubernetesConfiguration(m.KubernetesConfig, namespace)
+
+	if err := actionConfig.Init(cf, namespace, m.HelmDriver, debug); err != nil {
 		return nil, err
 	}
 
 	return actionConfig, nil
 }
 
+func getKubernetesConfiguration(kc KubernetesConfig, namespace string) *genericclioptions.ConfigFlags {
```

Comment:
Don't know if I really like this pattern here, looks a bit odd to me, but then again I wasn't able to find a better approach to copy over some of the "globals" into a new `ConfigFlags` struct instance.